pkg/pillar/types/diskmetrics.go

```diff
@@ -30,6 +28,12 @@ var ReportDirPaths = []string{
 	PersistDir + "/status",
 	PersistDir + "/certs",
 	PersistDir + "/checkpoint",
+	PersistDir + "/agentdebug",
+	PersistDir + "/containerd-system-root",
+	PersistDir + "/ingested",
+	PersistDir + "/netdump",
+	PersistDir + "/patchEnvelopesCache",
+	PersistDir + "/pubsub-large",
```

Comment:
How do we choose what to include in statistics?
I have some other dirs in mind that are not mentioned here. For example, `snapshots`. But it contains only JSON files and should not cause a lot of disk usage. At the same time, I see a status here that also contains only JSON files, as far as I remember.
And here is a disk usage of a random board I SSHed to:
```
16.0K	/persist/SMART_details.json
16.0K	/persist/SMART_details_previous.json
268.0K	/persist/agentdebug
224.0K	/persist/certs
48.0K	/persist/checkpoint
8.0K	/persist/clear
316.0K	/persist/containerd-system-root
4.0K	/persist/ingested
96.0K	/persist/installer
8.0K	/persist/log
16.0K	/persist/lost+found
1.5M	/persist/netdump
11.7M	/persist/newlog
8.0K	/persist/patchEnvelopesCache
8.0K	/persist/patchEnvelopesUsageCache
300.0K	/persist/pubsub-large
212.0K	/persist/status
4.0K	/persist/tmp
3.2G	/persist/vault
``` 
So the installer and the patch envelopes dirs are also the candidates, I would say...

Reply:
@OhmSpectator The algorithm I used was to look at a test machine and see which directories are not covered, plus guessing that one of the PathEnvelopes* was old (which might be incorrect and we should include both @uncleDecart ?)
I didn't see there was the installer dir (perhaps because it was installed years ago).
I didn't see a /persist/snapshots or a /persist/vault/snapshots - which path is it?

I see that we're missing cleanup of *.json.bak files and some files (such as the bak files in pubsub-large) can be quite big.
So good to have visibility into all the current directories in /persist.
(And we need to add some code to cleanup foo.json.bak when we delete foo.json but that is a separate issue.)

Reply:
@erik, snapshot-related JSON files are stored in `/persist/snapshots`.
In general, we could use the constants defined here: https://github.com/lf-edge/eve/blob/d00af13ec84b1fa5f67fadaeb20d7ff86d14e60b/pkg/pillar/types/locationconsts.go#L8-L111
It would be a better approach not to forget anything, I think.

Reply:
I've updated the list of directories using the defined constants where they exists.